http/src/main/resources/reference.conf

```diff
@@ -71,7 +71,9 @@ dataproc {
 
   customDataprocImage = "projects/broad-dsp-gcr-public/global/images/leo-dataproc-image-2-0-51-debian10-2023-04-12-19-18-34"
 
-  dataprocReservedMemory = 6g
+  # The ratio of memory allocated to spark. 0.8 = 80%.
```

Comment:
Is this documented anywhere? Would be helpful for the next dev working on it

Reply:
What specifically? The reasoning?

Reply:
All good, if it is not an official hail default documented somewhere this will do